From 61d00ebdbaa5dff6654cbbfa74ced29c414c8b4d Mon Sep 17 00:00:00 2001
From: IsaiahPatton <isaiahpatton1@gmail.com>
Date: Wed, 7 Jun 2017 19:33:18 -0400
Subject: [PATCH 06/12] replace rainbow with train

---
 .../mixins/MixinDedicatedServer.java               | 28 ++++++++++++----------
 1 file changed, 16 insertions(+), 12 deletions(-)

diff --git a/rainbow/src/main/java/org/projectrainbow/mixins/MixinDedicatedServer.java b/rainbow/src/main/java/org/projectrainbow/mixins/MixinDedicatedServer.java
index ebfea10..820f733 100644
--- a/rainbow/src/main/java/org/projectrainbow/mixins/MixinDedicatedServer.java
+++ b/rainbow/src/main/java/org/projectrainbow/mixins/MixinDedicatedServer.java
@@ -34,20 +34,24 @@ public class MixinDedicatedServer {
         _DiwUtils.ServerStartTime = System.currentTimeMillis();
         _DiwUtils.Startup();
 
+        // ProjectRails: start
         System.out.println(" ");
-        System.out.println("                  ___________________________");
-        System.out.println("           _,----'  _______________________  `----._");
-        System.out.println("        ,-'  __,---'  ___________________  `---.__  `-.");
-        System.out.println("     ,-'  ,-'  __,---'  _______________  `---.__  `-.  `-.");
-        System.out.println("   ,'  ,-'  ,-'  __,---'                `---.__  `-.  `-.  `.");
-        System.out.println("  /  ,'  ,-'  ,-'                               `-.  `-.  `.  \\");
-        System.out.println(" / ,'  ,' ,--' ____        _       __              `--. `.  `. \\");
-        System.out.println("| /  ,' ,'    /    \\____ _( )___  / /_  ____ _      __ `. `.  \\ |");
-        System.out.println("             / /_/ / __ `/ / __ \\/ __ \\/ __ \\ | /| / /");
-        System.out.println("            / _, _/ /_/ / / / / / /_/ / /_/ / |/ |/ / ");
-        System.out.println("           /_/ |_|\\__,_/_/_/ /_/_.___/\\____/|__/|__/  ");
+        System.out.println("               .---._");
+        System.out.println("           .--(. '  .).--.      . .-.");
+        System.out.println("        . ( ' _) .)` (   .)-. ( ) '-'");
+        System.out.println("       ( ,  ).        `(' . _)");
+        System.out.println("     (')  _________      '-'");
+        System.out.println("     ____[_________]                                         ________");
+        System.out.println("     \\__/ | _ \\  ||    ,;,;,,                               [________]");
+        System.out.println("     _][__|('')/__||  ,;;;;;;;;,   __________   __________  _| RAILS |_");
+        System.out.println("    /             | |____      | |          | |  ___     | |      ____|");
+        System.out.println("   (| .--.    .--.| |     ___  | |   |  |   | |      ____| |____      |");
+        System.out.println("   /|/ .. \\~~/ .. \\_|_.-.__.-._|_|_.-:__:-._|_|_.-.__.-._|_|_.-.__.-._|");
+        System.out.println("+=/_|\\ '' /~~\\ '' /=+( o )( o )+==( o )( o )=+=( o )( o )+==( o )( o )=+=");
+        System.out.println("='=='='--'==+='--'===+'-'=='-'==+=='-'+='-'===+='-'=='-'==+=='-'=+'-'jgs+");
+        System.out.println("Booted Project Rails version " + _DiwUtils.version + " Upstream version " + _DiwUtils.upstream_version);
         System.out.println(" ");
-        System.out.println("Rainbow Mod " + Bootstrap.minecraftVersion + "#" + Bootstrap.buildNumber);
+        // ProjectRails: end
     }
 
     @Redirect(method = "init", at = @At(value = "INVOKE", target = "net.minecraft.server.dedicated.PropertyManager.getStringProperty(Ljava/lang/String;Ljava/lang/String;)Ljava/lang/String;"))
-- 
2.13.0.windows.1

