From 84c6d2175ddb7b6289076868ec685122cb58fc8c Mon Sep 17 00:00:00 2001
From: IsaiahPatton <isaiahpatton1@gmail.com>
Date: Wed, 31 May 2017 12:32:01 -0400
Subject: [PATCH 14/14] add offline ban api

---
 .../java/org/projectrainbow/OfflinePlayerWrapper.java    | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/rainbow/src/main/java/org/projectrainbow/OfflinePlayerWrapper.java b/rainbow/src/main/java/org/projectrainbow/OfflinePlayerWrapper.java
index 2d2d401..eafc767 100755
--- a/rainbow/src/main/java/org/projectrainbow/OfflinePlayerWrapper.java
+++ b/rainbow/src/main/java/org/projectrainbow/OfflinePlayerWrapper.java
@@ -5,6 +5,8 @@ import PluginReference.*;
 import joebkt._JOT_OnlineTimeEntry;
 import joebkt._SerializableLocation;
 import net.md_5.bungee.api.chat.BaseComponent;
+import net.minecraft.entity.player.EntityPlayerMP;
+
 import org.projectrainbow.commands._CmdNameColor;
 
 import java.net.SocketAddress;
@@ -545,4 +547,18 @@ public class OfflinePlayerWrapper implements MC_Player {
                 location.dimension, location.yaw, location.pitch
         ));
     }
+
+    // ProjectRails: start ban api
+    @Override
+    public boolean isBanned() {
+        net.minecraft.entity.player.EntityPlayerMP plr = (EntityPlayerMP) ServerWrapper.getInstance().getOfflinePlayerByUUID(getUUID());
+        return _DiwUtils.getMinecraftServer().getPlayerList().getBannedPlayers().isBanned(plr.getGameProfile());
+    }
+
+    @Override
+    public void setBanned(boolean ban) {
+        net.minecraft.entity.player.EntityPlayerMP plr = (EntityPlayerMP) ServerWrapper.getInstance().getOfflinePlayerByUUID(getUUID());
+        _DiwUtils.getMinecraftServer().getPlayerList().getBannedPlayers().addEntry(new net.minecraft.server.management.UserListBansEntry(plr.getGameProfile()));
+    }
+    // ProjectRails: end ban api
 }
-- 
2.13.0.windows.1

